.ci/validation.sh

```diff
@@ -15,12 +15,25 @@ eclipse-cs)
                            | sed "s/-SNAPSHOT//")
   cd ../
   cd sevntu-checks
+  SEVNTU_ECLIPSE_CS_VERSION=$(mvn -e -q -Dexec.executable='echo' \
+                   -Dexec.args='${checkstyle.eclipse-cs.version}' \
+                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo "sevntus eclipse cs version: "$SEVNTU_ECLIPSE_CS_VERSION
   mvn -B -e clean install -Pno-validations
   cd ..
   checkout_from "https://github.com/checkstyle/eclipse-cs.git"
   cd .ci-temp/eclipse-cs/
   echo "Eclipse-cs tag: "$ECLIPSECS_TAG_NAME
   git checkout $ECLIPSECS_TAG_NAME
+  ECLIPSE_CS_CS_VERSION=$(mvn -e -q -Dexec.executable='echo' \
+                   -Dexec.args='${maven.sevntu.checkstyle.plugin.checkstyle.version}' \
+                   --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
```

Comment:
Command is failing.

````
++ mvn -e -q -Dexec.executable=echo '-Dexec.args=${maven.sevntu.checkstyle.plugin.checkstyle.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec
+ ECLIPSE_CS_CS_VERSION='[ERROR] Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.3.1:exec (default-cli) on project net.sf.eclipsecs.parent: Command execution failed.: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal org.codehaus.mojo:exec-maven-plugin:1.3.1:exec (default-cli) on project net.sf.eclipsecs.parent: Command execution failed.
````

Reply:
POM variable is only available on 10.3.4 . We can't use this to test the current issue.

Reply:
Command fails with 10.4.0 as well. I am not sure why.